toxcore/tox.h

```diff
@@ -250,7 +241,8 @@ uint32_t tox_nospam_size(void);
  * @brief The size of a Tox address in bytes.
  *
  * Tox addresses are in the format
- * `[Public Key (TOX_PUBLIC_KEY_SIZE bytes)][nospam (4 bytes)][checksum (2 bytes)]`.
+ * `[Public Key (TOX_PUBLIC_KEY_SIZE bytes)][nospam (4 bytes)]
```

Comment:
@iphydf cimple errors on this line

```
Processing toxcore/tox.h
toxcore/tox.h:244:4: Parse error near lexical error: "`"; expected end of file
Error: Process completed with exit code 1.
```

Reply:
It doesn't expect ` to run beyond a single line.

Reply:
Want me to revert it to being a single line, or do we want cimple to learn matching ` across lines?

Reply:
Revert for now, I don't have time to teach cimple this week.

Reply:
Done